README.md

```diff
@@ -129,6 +129,16 @@ clear_cache("21311-0001")  # only deletes the data for the object with the speci
 clear_cache()  # deletes the complete cache
 ```
 
+### Full documentation
+
+The full documentation currently is not hosted online and has to be build locally. This can is achieved by running
```

Comment:
Is it not possible to link to the created artifact as latest cicd run, pointing to html files which should be rendered? At least that is how I did it in another project but with Atlassian Bamboo and not Github but I think the artifact should be available as resource? Is this artifact a folder containing html or always an archive? Can we configure this?

Reply:
Linking to the Actions / Workflow would be possible but unspecific, linking to a specific artifact will get outdated quickly - would thus advocate for not linking it somewhere since I'm not aware of good and persistent solution

Reply:
Of course, I meant linking the latest, normally cicd provides the possibility to link the latest successful run, I will have a look

Reply:
But doesn't matter as we decided to do the github pages variant, so please continue!